packages/flutter/test/foundation/leak_tracking.dart

```diff
@@ -169,6 +169,11 @@ class LeakCleaner {
 
   /// Returns true if [leak] should be reported as failure.
   bool _shouldReportLeak(LeakType leakType, LeakReport leak, Map<(String, LeakType), int> countByClassAndType) {
+    if (leakType == LeakType.notDisposed && config.allowAllNotDisposed) {
```

Comment:
flutter style is to use switch/case on enums like LeakType instead of comparing single values so the analyzer reminds us to update these when a new enum value is added